packages/mobile/package.json

```diff
@@ -109,7 +109,7 @@
     "react-native-email-link": "^1.9.1",
     "react-native-exit-app": "^1.1.0",
     "react-native-flag-secure-android": "git://github.com/kristiansorens/react-native-flag-secure-android#e234251",
-    "react-native-fs": "^2.16.6",
+    "react-native-fs": "git+https://github.com/celo-org/react-native-fs#aa6db0f",
```

Comment:
maybe we can add a short comment here why we are forking it

Reply:
Since it's a json I don't think we can leave comments :( I can add a link to this PR on the README of the forks so if someone's wondering they can go there to see why we forked them.